.circleci/config.yml

```diff
@@ -981,7 +981,7 @@ jobs:
       - slack-notify-failure-if-any
 
   build-docs:
-    executor: xcode-latest
+    executor: 13.4.1
```

Comment:
We don't have a 13.4.1 executor. Please, check the CIrcleCI doc to assign corresponding macOS/Xcode image

Reply:
Updated in latest